rasa/core/brokers/kafka.py

```diff
@@ -7,6 +7,7 @@
 from rasa.core.brokers.broker import EventBroker
 from rasa.shared.utils.io import DEFAULT_ENCODING
 from rasa.utils.endpoints import EndpointConfig
+from rasa.shared.exceptions import KafkaProducerInitializationError
```

Comment:
I'd keep this exception in the `Kafka` module. I don't think there is a reason to use this exception outside of the `Kafka` module.

Reply:
Ah yeah that makes sense, 👍